commands/publish_dafsa.py

```diff
@@ -0,0 +1,64 @@
+import requests
+from requests.exceptions import HTTPError
+import wget
```

Comment:
I used wget, an external library for handling downloads. It works for now but I think it may be better to write another downloader function that uses the requests module from the standard library. Should I write the new function or is this fine?

Reply:
I think it just takes a few lines to download using `requests` so that would be better indeed

Reply:
replaced with native `requests` module, let's hope it doesn't break :)